server/gc/src/main/java/org/apache/accumulo/gc/SimpleGarbageCollector.java

```diff
@@ -620,7 +620,7 @@ private HostAndPort startStatsService() {
       processor = new Processor<>(rpcProxy);
     }
     int[] port = getConfiguration().getPort(Property.GC_PORT);
-    HostAndPort[] addresses = TServerUtils.getHostAndPorts(this.opts.getAddress(), port);
+    HostAndPort[] addresses = TServerUtils.getHostAndPorts(getHostname(), port);
```

Comment:
I'm not sure this is right. There is a difference between the address used for the bind address, and the hostname and port that was previously returned from `getHostname()` that was stored in ZooKeeper for the service address.

Reply:
The AbstractServer set hostname=opts.getAddress() and provided a method getHostname() used by the extending server to get their host address.  Note that opts.getAddress() defaults to 0.0.0.0 when no address was provided on the command line.

Reply:
Okay, so I was confused. I had thought `getHostname()` returned the name lookup from the IP address, but apparently that is done in `TServerUtils` much, much later. So, you're right... this is effectively the same... and there's no reason to use the opts here when `super.getHostname()` is sufficient. The method is, perhaps, badly named and misleading, but that can be addressed when the `--address` stuff is addressed, in future.

Reply:
I thought about changing it to getAddress() but as you said ...later.